### ISSUE TITLE
Tool generating empty lines for some resources

### ISSUE THREAD

**Issue by GeertvanHorrik** [dotnet/ResXResourceManager#504]
**Describe the bug**

When the tool is generating resx files, some resources get new lines added in the xml file containing 4 spaces:

```
  <data name="MyApp.MyResource" xml:space="preserve">
    <value>Some translation\nitem</value>
    
    
    
    
    
    
    
    
    
    
    
    
    
    
    
    
    
    
  </data>
```

This always causes git to see new changes while there are none.

**To Reproduce**

Have not yet found a reliable way to reproduce, but will update the ticket after research.

**Comment by tom-englert** [comment]
@GeertvanHorrik did you find anything yet?

**Comment by GeertvanHorrik** [comment]
Sorry, forgot about this one...

I did find something interesting. The items that are causing the empty lines do have this:

```
<note from="resx">Choice will be shown after this</note>
<note from="MultilingualBuild" annotates="source" priority="2">Choice will be shown after this</note>
```

I think it's because in the resx editor (of Visual Studio), I added the comment. Then **I think** the Multilingual Application Toolkit (MAT) added the `MultilingualBuild` element.

Have forked the project, trying to see if I can provide a PR.

**Comment by GeertvanHorrik** [comment]
This line looks suspicious:

https://github.com/dotnet/ResXResourceManager/blob/60835ac1ed01383239e1559e8660aa157a915c12/src/ResXManager.Model/XlfExtensions.cs#L120

**Comment by GeertvanHorrik** [comment]
Cannot yet reproduce, but created a PR here with the first unit tests:

https://github.com/dotnet/ResXResourceManager/pull/519

**Comment by tom-englert** [comment]
Fixed by #519
